packages/lumx-core/src/scss/components/text/_index.scss

```diff
@@ -3,23 +3,26 @@
    ========================================================================== */
 
 .#{$lumx-base-prefix}-text {
+    white-space: var(--lumx-text-white-space);
+
     &--is-truncated {
+        --lumx-text-white-space: "nowrap";
```

Comment:
```suggestion
        --lumx-text-white-space: nowrap;
```

Reply:
Oopsy, I've changed it and remove the quotes